bigquery_etl/query_scheduling/templates/airflow_dag.j2

```diff
@@ -13,44 +13,57 @@ default_args = {{
 
 with DAG('{{ name }}', default_args=default_args{%+ if schedule_interval != None -%}, schedule_interval={{ schedule_interval | format_timedelta | format_schedule_interval }}{%+ endif -%}) as dag:
 {% for task in tasks | sort(attribute='task_name') %}
-    {{ task.task_name }} = bigquery_etl_query(
-        task_id='{{ task.task_name }}',
-        destination_table={%+ if task.destination_table -%}'{{ task.destination_table }}'
-                          {%+ else -%}None
-                          {%+ endif -%},
-        dataset_id='{{ task.dataset }}',
-        project_id='{{ task.project }}',
-        owner='{{ task.owner }}',
-        {%+ if task.email | length > 0 -%}
-        email={{ task.email | sort }},
-        {%+ endif -%}
-        {%+ if task.start_date -%}
-        start_date={{ task.start_date | format_date | format_repr }},
-        {%+ endif -%}
-        {%+ if task.date_partition_parameter == None or task.date_partition_parameter is string -%}
-        date_partition_parameter={{ task.date_partition_parameter | format_optional_string }},
-        {%+ endif -%}
-        depends_on_past={{ task.depends_on_past }},
-        {%+ if task.arguments | length > 0 -%}
-        arguments={{ task.arguments }},
-        {%+ endif -%}
-        {%+ if task.parameters | length > 0 -%}
-        parameters={{ task.parameters }},
-        {%+ endif -%}
-        {%+ if task.multipart -%}
-        multipart={{ task.multipart }},
-        {%+ endif -%}
-        {%+ if task.sql_file_path -%}
-        sql_file_path={{ task.sql_file_path | format_repr }},
-        {%+ endif -%}
-        {%+ if task.priority -%}
-        priority_weight={{ task.priority }},
-        {%+ endif -%}
-        {%+ if task.allow_field_addition_on_date -%}
-        allow_field_addition_on_date={{ task.allow_field_addition_on_date | format_repr }},
-        {%+ endif -%}
-        dag=dag,
-    )
+    {% if not task.is_python_script -%}
```

Comment:
I'd prefer to remove the `not` here and put the python script case first to make this easier to read.